addon/templates/components/power-select-infinity/trigger.hbs

```diff
@@ -12,8 +12,12 @@
   onkeydown={{action "handleKeydown"}}
   onmousedown={{action "stopPropagation"}}>
 {{#if select.loading}}
-    <span class="ember-power-select-typeahead-loading-indicator">
-        <div class="double-bounce1"></div>
-        <div class="double-bounce2"></div>
-    </span>
+    {{#if loadingComponent}}
+        {{component loadingComponent}}
+    {{else}}
+        <span class="ember-power-select-typeahead-loading-indicator">
+            <div class="double-bounce1"></div>
+            <div class="double-bounce2"></div>
+        </span>
+    {{/if}}
```

Comment:
@bakerac4 I was thinking this would be done similarly to how the other power-select child components are, where you create a default loading component, `power-select-infinity/loading`, and set it as the default value for `loadingComponent`.  Then you wouldn't need this if/else block at all, it would just be simply:

```hbs
{{component loadingComponent}}
```